oat-sa/tao-core
# [Feature][AUT-2048] Enable text to wrap pictures

# [AUT-2048](https://oat-sa.atlassian.net/browse/AUT-2048)

## Require PRs
* https://github.com/oat-sa/tao-core-ui-fe/pull/446

## Changes
* Added migration to remove `Image Alignment` status flag
* `SetImageAligmentConfig` installation script was changed:
  * Will enable `Image Alignment` plugin by default
  * Added public constant `PARAM_MEDIA_ALIGNMENT` for `mediaAlignment` parameter
* `SetImageAligmentConfig` installation script was removed from manifest file, as we do not need this flag anymore

## How to test
* Installation
  1. Do a fresh instance installation
  2. Check that `Image Alignment` plugin is enabled
* Update
  1. Check that `Image Alignment` plugin is disabled
  2. Update an existing instance
  3. Check that `Image Alignment` plugin is enabled
